migrations/versions/2022b01697e4_.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""empty message
+
+Revision ID: 2022b01697e4
+Revises: f19732b1101b
+Create Date: 2022-08-24 22:47:51.007940
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+import app
+import app.extensions
+
+# revision identifiers, used by Alembic.
+revision = '2022b01697e4'
+down_revision = 'f19732b1101b'
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('mission_task_user_assignment', schema=None) as batch_op:
+        batch_op.add_column(
```

Comment:
Can't this be done as a two step operation, create it as nullable=True. 
But then I guess, what user guid do you put here? Not sure

Reply:
exactly - this is the problem we have had before with nullable=False for _foreign keys_.  (if it was just a simple value like int or string we could do as you describe:  nullable=True, set to some "acceptable" dummy value, then nullable=False.)  but when its pointing to something like a user, not so much.